recipes/default.rb

```diff
@@ -16,6 +16,12 @@
 # limitations under the License.
 #
 
+directory  "/var/chef/handlers" do
```

Comment:
Instead of a hardcoded directory name, this should be an attribute or a `Chef::Config` option. Looking at the `cookbook_file` resource that our Chef runs are failing on, they're using a value of `node['chef_handler']['handler_path']`, so we should also use that for this directory resource like so:

`directory node['chef_handler']['handler_path'] do`